README.md

```diff
@@ -309,7 +309,8 @@ See the [API documentation](./docs/api.md) for all endpoints.
 - [PyOllaMx](https://github.com/kspviswa/pyOllaMx) - macOS application capable of chatting with both Ollama and Apple MLX models.
 - [Claude Dev](https://github.com/saoudrizwan/claude-dev) - VSCode extension for multi-file/whole-repo coding
 - [Cherry Studio](https://github.com/kangfenmao/cherry-studio) (Desktop client with Ollama support)
-  
+- [ConfiChat](https://github.com/1runeberg/confichat) (Lightweight, standalone, multi-platform, and privacy focused LLM chat interface with optional encryption)
```

Comment:
```suggestion
```

Possible to add it to one section for now? I think the Terminal section might be best?

Reply:
It's not a terminal client though. It's a standalone gui. It has both Mobile and Desktop versions. 

Is there an issue having the app on both sections (Desktop & Mobile)? I have pre-compiled builds for all platforms in the repo if you need to verify :)

Reply:
Ah, sorry I misread the diff! This is okay for now

Reply:
Thanks Jeffrey! 🎉